### PR TITLE
client to server connection working!

### PR DESCRIPTION
**Pull request by stevens-works**

### CODE REVIEW

**Review by Siege211 — COMMENTED** [review]
LGTM

**Comment by stevens-works** [comment]
@Siege211 thanks for reviewing. ill go ahead and merge 👍

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/node_modules
+/client/dist/bundle.js
```
Comment (therobinkim) [inline .gitignore:2]: FYI you added bundle.js to your gitconfig, but you still committed it; by removing your bundle.js in the future, you can make your PR even smaller (by 296 lines of code, in this case)

File: server/index.js [hunk 2]
```diff
@@ -9,15 +10,20 @@ app.use(bodyParser.urlencoded({ extended: true }));
...
-// GET request to db
-app.get('/api/songs-info', (req, res) => {
-  if (err) {
-    console.log('ERR: fix GET server routing');
-  } else {
-    res.send(JSON.stringify(data));
-  }
+// GET request to render index.html
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname, '../dist/index.html'));
```
Comment (therobinkim) [inline server/index.js:15]: this middleware is unnecessary; you're already using `express.static`

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    path: DIST_DIR
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?/,
+        include: SRC_DIR,
+        loader: 'babel-loader',
+        query: {
+          presets: ['react', 'es2015']
```
Comment (therobinkim) [inline webpack.config.js:18]: use `env`, not `es2015`

**Pull request merged**